guides/github.md

```diff
@@ -0,0 +1,130 @@
+# How to secure GitHub repositories
+
+It is recommended to turn on 2FA for most repositories, especially if building
+a package that is a dependency of multiple other packages.
+
+At least for critical packages, administrators should enable branch
+protection. Require CI to pass before merging to the main branch. Allow only
+repository owners to merge PRs.
+
+> [!WARNING]
+> It is recommended to run workflows only after the PR has been reviewed. The
+> "Require approval for all outside collaborators" setting is the recommended
+> one. However, this can cause friction in repositories with a high number of
+> PRs but small number of contributors. In this case, if all contributors are
+> trusted, "Require approval for first-time contributors" is a valid option,
+> but dedicate more efforts to separate sensitive steps into workflows that
+> can only run after approval.
+
+It is ideal to require PR reviews before merging. For security critical
+packages, having at least two reviewers is ideal, as this alleviates the risk
+of sock-puppet accounts.
+
+To keep dependencies up to date, it is advisable to enable
+[Dependabot][dependabot] or [Renovate][renovate] or a similar tool which will
```

Comment:
Both tools don't support Haskell afaik, so why are they recommended?

- https://github.com/dependabot/dependabot-core/issues/2745
- https://github.com/renovatebot/renovate/issues/8187

Reply:
I was wrong. I thought the support already existed. I can remove them and add them later when we have support.

But I'd still like to have a section about updating dependencies and I would still recommend these for updating github actions (and other non-Haskell packages).

For now, I'm going to add a mention that support for Haskell is missing at the moment.

I will also recommend https://github.com/nomeata/haskell-bounds-bump-action